### ISSUE TITLE
IPv6 support

### ISSUE THREAD

**Issue by unixfox** [rackerlabs/spot-roadmap#22]
It would be great to have IPv6 support in the Kubernetes cluster.

At the very least, on the Kubernetes nodes side. Load balancers could come later (if they do not already support ipv6?)

See here why IPv6 is important: https://old.reddit.com/r/explainlikeimfive/comments/1k45y7/eli5_what_is_ipv6_and_why_is_it_so_important/

Also, this allows to communicate with services that are only accessible in IPv6.

**Comment by unixfox** [comment]
It seems like nodes have an IPv6 address but you can't use it in pods. The CNI doesn't support IPv6.

**Comment by sirishkr** [comment]
Tracking for prioritization in Spring 2026
